### PR TITLE
Little english improvements

### PR DESCRIPTION
**Pull request by angristan**

### CODE REVIEW

**Review by acid-chicken — APPROVED** [review]
:shipit:

**Comment by syuilo** [comment]
若干編集合戦っぽい(冠詞が付いたり消えたり)

File: README.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ Please see the [Contribution Guide](./CONTRIBUTING.md).
 ----------------------------------------------------------------
 > Copyright (c) 2014-2019 syuilo
 
-Misskey is open-source software licensed under the [GNU AGPLv3](LICENSE).
+Misskey is an open-source software licensed under the [GNU AGPLv3](LICENSE).
```
Comment (natarii) [inline README.md]: The original text here is okay. "a[n] software" is grammatically incorrect.

**Review by natarii — CHANGES_REQUESTED** [review]
The original phrasing of "Misskey is open-source software" is okay.

File: README.md (hunk 1 above)
Comment (angristan) [reply README.md]: Thanks!

(For reference: https://forum.wordreference.com/threads/software-vs-a-software.3216740/)

**Review by AyaMorisawa — APPROVED** [review]

**Review by acid-chicken — APPROVED** [review]

**Comment by syuilo** [comment]
Thank you guys 👍

**Pull request merged**